spec/components/placement/organisation_switcher_component_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+RSpec.describe Placement::OrganisationSwitcherComponent, type: :component do
+  subject(:component) do
+    described_class.new(user:, organisation:)
+  end
+
+  let(:organisation) { create(:placements_school) }
+
+  context "when the user is a support user" do
+    let(:user) { create(:placements_support_user) }
+
+    it "renders the organisation switcher" do
+      render_inline(component)
+
+      within(".content-header") do
+        expect(page).to have_content(organisation.name)
+        expect(page).to have_link(
+          text: "Change organisation",
+          href: public_send("placements_support_organisations_path"),
+        )
+      end
```

Comment:
It definitely doesn't feel good to be using `within(".content-header")` here. I think the component should stand alone from the layout section it gets injected into (as commented above).